overlord/snapstate/snapstate.go

```diff
@@ -1909,6 +1933,22 @@ func Remove(st *state.State, name string, revision snap.Revision, flags *RemoveF
 		if tp, _ := snapst.Type(); tp == snap.TypeApp && removeAll {
 			ts, err := AutomaticSnapshot(st, name)
 			if err == nil {
+				sz, err := EstimateSnapshotSize(st, name, nil)
+				if err != nil {
+					return nil, err
+				}
+				// require 5Mb extra
```

Comment:
I understand this is an estimation but could you please include some rationale about how this was computed?

Reply:
There isn't really any deep reason for that, see earlier discussion on this PR.